python/ray/data/context.py

```diff
@@ -154,6 +154,10 @@ class ShuffleStrategy(str, enum.Enum):
 
 DEFAULT_ENABLE_GET_OBJECT_LOCATIONS_FOR_METRICS = False
 
+DEFAULT_DOWNSTREAM_CAPACITY_OUTPUTS_RATIO: int = env_float(
+    "RAY_DATA_DOWNSTREAM_CAPACITY_OUTPUTS_RATIO", 25.0
+)
```

Comment:
Can you add a note explaining how this was chosen? Don't think it'll be obvious to someone who reads this an year from now